core/tests/test_utils.py

```diff
@@ -143,12 +158,26 @@ class TestBase(unittest.TestCase):
     }
 
     def _get_unicode_test_string(self, suffix):
+        """ Supply a test string that has unicode characters in it.
+
+        Args:
+            suffix: str. The suffix to UNICODE_TEST_STRING.
+
+        Returns:
+            str. A string UNICODE_TEST_STRING with suffix 'suffix'.
```

Comment:
A string that contains unicode characters and ends with the given suffix.

(Try not to put too much implementation detail in the docstrings. Rather, document the interface -- i.e., what a client who treats this as a "black box" would care about.)